apps/studio/components/interfaces/Organization/BillingSettings/Restriction.tsx

```diff
@@ -97,7 +97,11 @@ export const Restriction = () => {
             </p>
             <div className="flex items-center gap-x-2 mt-3">
               <Button asChild key="upgrade-button" type="default">
-                <Link href={`/org/${org?.slug}/billing?panel=subscriptionPlan`}>Upgrade plan</Link>
+                <Link
+                  href={`/org/${org?.slug}/billing?panel=subscriptionPlan&source=gracePeriodStarted`}
```

Comment:
```suggestion
                  href={`/org/${org?.slug}/billing?panel=subscriptionPlan&source=fairUseGracePeriodStarted`}
```

Reply:
Adding fair use here, just to make clear these are from the fair use / service restriction notes. Feel free to change if you can think of a better name